src/.nuget/NuGetRestore.ps1

```diff
@@ -1,5 +1,6 @@
 $NuGetExe = "$PSScriptRoot\NuGet.exe"
 
+& $NuGetExe install xunit.runner.console -version 2.0.0 -OutputDirectory "$PSScriptRoot\..\..\packages"
```

Comment:
Do the analyzer unit tests work with the RTM runner or are they running with no results/no output (as I would expect)?